docs/source/channel_update_tutorial.rst

```diff
@@ -4,38 +4,30 @@ Adding an Org to a Channel
 .. note:: Ensure that you have downloaded the appropriate images and binaries
           as outlined in :doc:`install` and :doc:`prereqs` that conform to the
           version of this documentation (which can be found at the bottom of the
-          table of contents to the left). In particular, your version of the
-          ``fabric-samples`` folder must include the ``eyfn.sh`` ("Extending
-          Your First Network") script and its related scripts.
-
-This tutorial serves as an extension to the :doc:`build_network` (BYFN) tutorial,
-and will demonstrate the addition of a new organization -- ``Org3`` -- to the
-application channel (``mychannel``) autogenerated by BYFN. It assumes a strong
-understanding of BYFN, including the usage and functionality of the aforementioned
-utilities.
-
-While we will focus solely on the integration of a new organization here, the same
-approach can be adopted when performing other channel configuration updates (updating
-modification policies or altering batch size, for example). To learn more about the
-process and possibilities of channel config updates in general, check out
-:doc:`config_update`). It's also worth noting that channel configuration updates like
-the one demonstrated here will usually be the responsibility of an organization admin
-(rather than a chaincode or application developer).
+          table of contents to the left).
+
+This tutorial extends the Fabric test network by adding a new organization
+-- Org3 -- to an application channel.
```

Comment:
Rather than putting that at top, I am going rephrase the cert generation section:
First, we are going to create the certificates and keys for the Org3 peer, along
with an application and admin user. Because we are updating an example channel,
we are going to use the ``cryptogen`` tool instead of using a Certificate Authority.
The following command uses ``cryptogen``  to read the ``org3-crypto.yaml`` file
and generate the Org3 crypto material in a new ``org3.example.com`` folder: